database/index.js

```diff
@@ -1,214 +1,149 @@
+    orderLabel = [['date', 'ASC']];
+  } else if (sort === 'helpful') {
+    orderLabel = [['helpfulness', 'ASC']];
+  } else if (sort === 'relevant') {
+    orderLabel = [['rating', 'ASC']];
   }
-}, {
-  sequelize: sequelize
-})
-
-const Info = sequelize.define('info', {
-  id: {
-    type: DataTypes.INTEGER,
-    primaryKey: true
-  },
-  product_id: {
-    type: DataTypes.INTEGER
-  },
-  rating: {
-    type: DataTypes.INTEGER
-  },
-  date: {
-    type: DataTypes.BIGINT
-  },
-  summary: {
-    type: DataTypes.TEXT
-  },
-  body: {
-    type: DataTypes.TEXT
-  },
-  recommended: {
-    type: DataTypes.BOOLEAN
-  },
-  reported: {
-    type: DataTypes.BOOLEAN
-  },
-  reviewer_name: {
-    type: DataTypes.TEXT
-  },
-  reviewer_email: {
-    type: DataTypes.TEXT
-  },
-  response: {
-    type: DataTypes.TEXT
-  },
-  helpfulness: {
-    type: DataTypes.INTEGER
-  }
-}, {
-  sequelize: sequelize
-});
-
-const Info_photos = sequelize.define('info_photos', {
-  id: {
-    type: DataTypes.INTEGER,
-    primaryKey: true,
-    allowNull: false
-  },
-  review_id: {
-    type: DataTypes.INTEGER,
-    allowNull: false
-  },
-  url: {
-    type: DataTypes.STRING
-  }
-}, {
-  sequelize: sequelize
-});
-
-const Characteristics_data = sequelize.define('char_data', {
-  id: {
-    type: DataTypes.INTEGER,
-    primaryKey: true,
-    allowNull: false
-  },
-  product_id: {
-    type: DataTypes.INTEGER,
-    allowNull: false
-  },
-  name: {
-    type: DataTypes.STRING
-  }
-}, {
-  sequelize: sequelize
+  Info.findAll({
+    where: {
+      product_id: productId
+    },
+    order: orderLabel,
+    limit: count,
+  })
+    .then((data) => {
+      var promiseOfPhotos = data.map((entry) => {
+        return Info_photos.findAll({
+          where: {
+            review_id: entry.id,
+          }
+        })
+      });
+      Promise.all(promiseOfPhotos)
+        .then((result, err) => {
+          if (err) {
+            throw err;
+          } else {
+          for (var i = 0; i < result.length; ++i) {
+            data[i].dataValues['photos'] = result[i];
+            data[i].dataValues['review_id'] = data[i].dataValues.id;
+            delete data[i].dataValues['id'];
+            delete data[i].dataValues['product_id'];
+          }
+          final['results'] = data;
+          res.send(final);
+          }
+        });
+    })
+    .catch((err) => {
+      res.send(err);
+    });
 });
 
-const Characteristics_reviews = sequelize.define('char_reviews', {
-  id: {
-    type: DataTypes.INTEGER,
-    primaryKey: true,
-    allowNull: false
-  },
-  characteristic_id: {
-    type: DataTypes.INTEGER,
-    allowNull: false
-  },
-  review_id: {
-    type: DataTypes.INTEGER,
-    allowNull: false
-  },
-  value: {
-    type: DataTypes.INTEGER,
-    allowNull: false
-  }
-}, {
-  sequelize: sequelize
+app.get('/reviews/meta', (req, res) => {
+  var productId = req.body.product_id;
+  var final = {
+    product_id: productId
+  };
+  Info.findAll({
```

Comment:
Consider placing this process within a function? To improve readability and the reasoning.
I think it would help with describing the purpose of this area of code. 
So who future readers could have an idea of what it is going to do.